spec/models/exceptions_spec.rb

```diff
@@ -0,0 +1,11 @@
+describe PinError do
+  it 'raises pin error with custom message' do
+    expect { raise described_class }.to raise_error described_class, subject.msg
+  end
+end
+
+describe GameOverError do
+  it 'raises game over error with custom message' do
+    expect { raise described_class }.to raise_error described_class, subject.msg
+  end
+end
```

Comment:
I don't think you really need these. Testing should be for behaviour, here your error class is just storage no logic, so I probably wouldn't have tests for it. (controversial, maybe)

Reply:
got it. I removed them and my coverage didn't change, so I agree with you. thanks